Integrations/integration-Intezer.yml

```diff
@@ -260,9 +274,12 @@ script:
   - name: file
     arguments:
     - name: file
-      required: true
       default: true
       description: Hash of the file to query. Supports SHA256
+      isArray: true
```

Comment:
use isArray only if the API support array

Reply:
mutually decided to keep it